### PR TITLE
dev-util/qdevicemonitor: fix Qt 5.11 specific issues

### PR DESCRIPTION
**Pull request by alopatindev**
Bug: https://bugs.gentoo.org/660932
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: dev-util/qdevicemonitor/files/qdevicemonitor-1.0.1-qt-5.11.patch [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+index ced04b3..6df67eb 100644
+--- a/qdevicemonitor/devices/DeviceFacade.cpp
++++ b/qdevicemonitor/devices/DeviceFacade.cpp
+@@ -151,8 +151,8 @@ void DeviceFacade::loadSettings(const QSettings& s)
+     if (darkTheme.isValid())
+     {
+         m_darkTheme = darkTheme.toBool();
+-        m_colorTheme = ColorTheme::create(m_darkTheme);
+     }
++    m_colorTheme = ColorTheme::create(m_darkTheme);
```
Comment (a17r) [inline dev-util/qdevicemonitor/files/qdevicemonitor-1.0.1-qt-5.11.patch]: How is this change related to Qt 5.11?

Comment (alopatindev) [reply dev-util/qdevicemonitor/files/qdevicemonitor-1.0.1-qt-5.11.patch]: Oops, you are right, it's unrelated. Thanks for noticing!

It fixes a critical bug though (a crash that happens for users with freshly installed application). It appeared in [this commit](https://github.com/alopatindev/qdevicemonitor/commit/8ff2006021e812530fcc1dd7a8faded9cbfe7040).

Should I rename the commit message, a patch filename and remove `-r1`?

Comment (a17r) [reply dev-util/qdevicemonitor/files/qdevicemonitor-1.0.1-qt-5.11.patch]: Ah, not a great example of git use for atomic fixes, otherwise I would have recommended to directly use the upstream commit. Still, please move the runtime fix to a separate patch file and include a short description with a link to a bug (if there is one) for reference so we know why we are doing this. the same is not required for the trivial qt-5.11 fix (was this submitted upstream though?).

The runtime fix indeed makes a revbump necessary.

Comment (alopatindev) [reply dev-util/qdevicemonitor/files/qdevicemonitor-1.0.1-qt-5.11.patch]: Both fixes are submitted to upstream but I've also split them into two commits. I don't have any reported bugs though.

I've split the patch and updated ebuild. Please recheck. Thanks!

File: dev-util/qdevicemonitor/qdevicemonitor-1.0.1-r1.ebuild [hunk 2]
```diff
@@ -27,6 +27,8 @@ RDEPEND="
 	app-pda/usbmuxd"
 DEPEND="${RDEPEND}"
 
+PATCHES=( "${FILESDIR}"/${P}-qt-5.11.patch "${FILESDIR}"/${P}-crash-after-fresh-install.patch )
```
Comment (a17r) [inline dev-util/qdevicemonitor/qdevicemonitor-1.0.1-r1.ebuild]: one line per patch please, but I'll fix that for you.

**Comment by a17r** [comment]
- Also dropped old ebuild in a separate commit
- Changed `Bug:` to `Closes:`
- Switched to desktop.eclass
